src/scalars/iso-date/DateTime.ts

```diff
@@ -33,7 +33,14 @@ export const GraphQLDateTimeConfig: GraphQLScalarTypeConfig<Date, Date> = /*#__P
       throw createGraphQLError(`DateTime cannot represent an invalid date-time-string ${value}.`);
     } else if (typeof value === 'number') {
       try {
-        return new Date(value);
+        // Find out if the value is in seconds or milliseconds
+        const dateInMilliseconds = Date.now();
```

Comment:
Instead of getting the current `Date`, we can check the number of digits with a fixed value like 
`value.toString().length === X`

Reply:
@yamatsafi Do you have time to fix this? I could help otherwise.

Reply:
I'd appreciate if you can :)

Reply:
This is future-proof instead of checking for 13-digits today which may not be true in a few years :-) I know it is a long way but still. @jeremyzahner Please feel free to add unit tests as @ardatan suggested.

Reply:
made changes, ready for review

Reply:
@ardatan ^